nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/ListS3.java

```diff
@@ -165,7 +165,7 @@ public class ListS3 extends AbstractS3Processor implements VerifiableProcessor {
 
     public static final PropertyDescriptor TRACKING_TIME_WINDOW = new PropertyDescriptor.Builder()
         .fromPropertyDescriptor(ListedEntityTracker.TRACKING_TIME_WINDOW)
-        .dependsOn(INITIAL_LISTING_TARGET, ListedEntityTracker.INITIAL_LISTING_TARGET_WINDOW)
+        .dependsOn(ListedEntityTracker.TRACKING_STATE_CACHE)
```

Comment:
Thanks for fixing the incorrect property dependency!

I would also change the order of the properties on the UI and use the same sequence as in the other List processors:

- `TRACKING_STATE_CACHE`
- `TRACKING_TIME_WINDOW`
- `INITIAL_LISTING_TARGET`

`INITIAL_LISTING_TARGET` can be `required=true` because it is only shown when it is required (due to `dependsOn`) and has a default value too. This way these 3 properties would appear consistently on the UI.